review-responses/first-barre.rst

```diff
@@ -0,0 +1,110 @@
+Dear Arnaud Barré,
+
+Thank you very much for lending your time and expertise for this review. We are
+happy that you have approved the article. We have submitted a second revision
+that addresses you comments. The responses are below:
+
+   However, it would be interesting to have a better comparison of the
+   different methods proposed to compute body segment inertial parameters using
+   direct and indirect methods. For example, the authors mentions the facility
+   of Yeadon's method compared to other studies, but the number of measurements
+   is large compared to others studies not cited, like de Leva (1996) or Dumas
+   et al. (2007). These indirect methods have also error as explained in Rossi
+   et al. (2013). What is the advantage of Yeadon's method compared to others?
+   A discussion on the limitations of the model proposed (joint center
+   accuracy, joint range, mass estimation, density scaling, etc.) would be also
+   of interest.
+
+We also think that a paper showing the disadvantages and advantages of these
+various models would be very interesting but we believe that this undertaking
+is out of the scope of this paper. To properly address this, would take much
+more space and time to complete than we have. We are leaving these discussions
```

Comment:
K I'm just going to remove this sentence. I think the paragraph is fine without it.